### PR TITLE
Dymo Scale

### PR DESCRIPTION
**Pull request by brentru**
Adding code and example code for interfacing with a [DYMO](http://www.dymo.com/en-US) scale with CircuitPython.

### CODE REVIEW

File: adafruit_dymoscale.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+    @property
+    def weight(self):
+        """Weight in grams"""
+        weight = self.get_scale_data()
+        if self.units == OUNCES:
+            self.units = 'oz'
+        elif self.units == GRAMS:
+            self.units = 'g'
+        return weight
```
Comment (ladyada) [inline adafruit_dymoscale.py]: convert it to grams no matter which one, so that weight is always consistantly unit'd

Comment (brentru) [reply adafruit_dymoscale.py]: Done!

File: adafruit_dymoscale.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        self.stable = data_bytes[2] & 0x4
+        self.units = data_bytes[3]
+        weight = data_bytes[5] + (data_bytes[6] << 8)
+        if data_bytes[2] & 0x1:
+            weight *= -1
+            print('Tare - press the tare button to reset the scale to zero.')
+        elif self.units == OUNCES:
+            if data_bytes[4] & 0x80:
+                data_bytes[4] -= 0x100
+                print('Tare - press the tare button to reset the scale to zero.')
```
Comment (ladyada) [inline adafruit_dymoscale.py]: dont print stuff in the library

File: adafruit_dymoscale.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        or data_bytes[11] != 0:
+            raise RuntimeError("Bad data capture")
+
+        # parse out the data_bytes
+        self.stable = data_bytes[2] & 0x4
+        self.units = data_bytes[3]
+        weight = data_bytes[5] + (data_bytes[6] << 8)
+        if data_bytes[2] & 0x1:
+            weight *= -1
+            print('Tare - press the tare button to reset the scale to zero.')
```
Comment (ladyada) [inline adafruit_dymoscale.py]: dont print

File: adafruit_dymoscale.py [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+        self.stable = None
+        self.units = None
+
+    @property
+    def weight(self):
+        """Weight in grams"""
+        weight = self.get_scale_data()
+        if self.units == OUNCES:
+            weight *= 28.35
+        self.units = 'g'
```
Comment (ladyada) [inline adafruit_dymoscale.py]: GRAMS

**Comment by ladyada** [comment]
ready for merge?

**Comment by brentru** [comment]
@ladyada  yep!

**Comment by ladyada** [comment]
rad. please pylint & bundle - @kattni can help!

**Pull request merged**